frontend/src/components/news-page/News.tsx

```diff
@@ -1,3 +1,43 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 
-export const News: React.FunctionComponent = () => <></>;
+export const News: React.FunctionComponent = () => {
+  const [articlesList, setArticlesList] = useState<any[]>([]);
+
+  useEffect(() => {
+    const url = "https://www.reddit.com/r/whales/top.json?limit=50&t=month";
+
+    const fetchData = async () => {
+      try {
+        const response = await fetch(url);
+        const json: any = await response.json();
+        const articles = json.data.children;
+        setArticlesList(
+          articles.filter((article: any) => article.data.is_video)
+        );
+      } catch (error) {
+        console.log("error", error);
+      }
+    };
+    fetchData();
+  }, []);
+
+  return (
+    <>
+      <h1>Whale content</h1>
+      {articlesList &&
+        articlesList.map((article) => (
+          <>
+            <p key={article.data.Id}>
+              {article.data.title.replace(/&amp;/g, "&")}
+            </p>
+            <video
+              src={article.data.media.reddit_video.fallback_url}
+              width="320"
+              height="320"
+              controls
+            />
+          </>
```

Comment:
I know an over-abundance of `<div>`s isn't so good, but we do want to semantically link this title to the video, and putting them both in the same `<div>` isn't a bad idea!

If you want to be _really_ semantic and accessible (and, let's be honest, who doesn't?) then you could look into [`<figure>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure) and [`<figcaption>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)!